LauncherInjector/main.cpp

```diff
@@ -310,6 +324,14 @@ int main(int argc, char* argv[])
 	}
 
 	{
+		if (!GetExePathWide(exePath, sizeof(exePath)))
+		{
+			MessageBoxA(
+				GetForegroundWindow(), "Failed getting game directory.\nThe game cannot continue and has to exit.",
+				"Northstar Launcher Error", 0);
+			return 1;
+		}
+
```

Comment:
This was moved up when I did the stubs, but it looks like a merge put it back here.